### PR TITLE
feat(cli): Add `blitz start` command #21

### PR DESCRIPTION
**Pull request by tsawan**
This is PR for issue #21. (adds `start` command with alias `s`)

### CODE REVIEW

**Comment by ryardley** [comment]
Also just checked and I think we want `start` to actually call `next dev` not `next start`

**Comment by flybayer** [comment]
Yes, you're right @ryardley. My bad on fake news 🤦‍♂

**Comment by tsawan** [comment]
will do.

**Comment by ryardley** [comment]
Hey @tsawan just hooking you in on what I have been cooking over [here](https://github.com/blitz-js/blitz/pull/45). Through this I am hoping you might have some good answers on how to test the long running process of `next dev` here that I can rebase off.

**Comment by tsawan** [comment]
@ryardley Sorry I didn't get that. Can you please elaborate a bit more?

**Comment by ryardley** [comment]
I guess I am wondering how we should test this.

**Comment by tsawan** [comment]
@ryardley will check that PR and get back to you.

**Comment by flybayer** [comment]
@tsawan looks like the only thing pending on this is `start --production`, right?

**Comment by tsawan** [comment]
@flybayer `start --production` is easy. I had issue with test code that is probably resolved by @ryardley. So if test code is working fine now, I will update the PR shortly.

**Comment by tsawan** [comment]
Please review @flybayer @ryardley

**Comment by tsawan** [comment]
not sure why jest --coverage is failing in ci. any idea?

File: packages/cli/test/commands/start.test.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import StartCmd from '../../src/commands/start'
+
+describe('Start command', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('runs next dev', async () => {
+    await StartCmd.run();
```
Comment (ryardley) [inline packages/cli/test/commands/start.test.ts]: Perhaps try passing in empty brackets here. I am guessing that might overwrite the argv array.

**Comment by ryardley** [comment]
🚀 lgtm

File: packages/cli/src/commands/start.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      description: 'runs a production build',
+      default: false,
+      allowNo: false,
+    })
+  }
+
+  async run() {
+    const {flags} = this.parse(Start)
+    if (flags.production) {
+      var cp = spawn('next', ['build'], {stdio: 'inherit'})
```
Comment (ryardley) [inline packages/cli/src/commands/start.ts]: Don't think it matters right now really and don't think it is worth talking about here but I would vote for some kind of linting rule to limit `var` use in the future.

**Comment by flybayer** [comment]
Closing in favor of #45

**Pull request closed without merging**